corehq/apps/tour/models.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from django.contrib.auth.models import User
+from django.db import models
+import json_field
+
+
+class GuidedTours(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.PROTECT,
+        unique=True
+    )
+    seen_tours = json_field.JSONField(
+        default={},
+    )
+
+    @classmethod
```

Comment:
Or you could use the `cls` arg rather than referencing the `GuidedTours` global.